framework/src/Volo.Abp.BlazoriseUI/Components/PageHeader.razor.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Components;
+using Volo.Abp.AspNetCore.Components.WebAssembly;
+
+namespace Volo.Abp.BlazoriseUI.Components
+{
+    public partial class PageHeader : ComponentBase
+    {
+        [Parameter]
+        public string Title { get; set; }
+
+        [Parameter]
+        public bool BreadcrumbShowHome { get; set; } = true;
+
+        [Parameter]
+        public bool BreadcrumbShowCurrent { get; set; } = true;
+
+        [Parameter]
+        public RenderFragment ChildContent { get; set; }
+
+        protected List<BreadcrumbItem> BreadcrumbItems { get; set; }
```

Comment:
Maybe also have a `[Parameter]` for this so it can be set declaratively through razor syntax.

Reply:
Done.